packages-ts/gauntlet-terra/src/commands/internal/terra.ts

```diff
@@ -22,8 +23,11 @@ export default abstract class TerraCommand extends WriteCommand<TransactionRespo
   provider: LCDClient
   contracts: string[]
   public codeIds: CodeIds
+
   abstract execute: () => Promise<Result<TransactionResponse>>
   abstract makeRawTransaction: (signer: AccAddress) => Promise<MsgExecuteContract>
+
+  simulateExecute: () => any
```

Comment:
Is this necessary?

Reply:
With [this change](https://github.com/smartcontractkit/chainlink-terra/pull/157/files#diff-48dc1bdb44afa06fa4ff99d7b4fef1f6dc28a090ead3796e0c5365bd711d0c29R24) shouldn't be necessary